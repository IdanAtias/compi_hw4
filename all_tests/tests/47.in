
F i=0 F;
while(true){
	i=i+1 F;
	if(i==8 F)
		break;
}

F done;
