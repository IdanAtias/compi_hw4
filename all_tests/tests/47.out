-- end scope --
-- end scope --
var:i:0:FAHRENHEIT
var:done:1:FAHRENHEIT
